src/components/FormValidator.js

```diff
@@ -38,7 +36,6 @@ class FormValidator {
       this._submitButton.disabled = true;
```

Comment:
You have the `reset` method  for that. Please, call it here not to duplicate the same code